staff/groups/los-orbitals/nutrilab/app/src/App.js

```diff
@@ -64,6 +64,11 @@ class App extends Component {
     this.props.history.push('/login')
   }
 
+   // Prepare the view to show the Login form and update the url path
+   goToDeleteProfile = () => {
+    this.props.history.push('/deleteprofile')
```

Comment:
this.props.history.push('/deleteprofile') no existe esa ruta, abajo has creado la ruta /delete, puede que pete por ello... ajjaja mañana lo miramos